browser-es6.js

```diff
@@ -21,5 +21,12 @@ module.exports = merge(base, {
   },
   plugins: [
     'react'
-  ]
+  ],
+  rules: {
+    'new-cap': 0,
```

Comment:
Ugh, come to think of it I ran into this exact case in Analytics.js. (Why couldn't they have just made this a static method...?)

This is a common case across our repos, can you move this into `browser.js` and add a note above it to the effect of, `// Emitter: component/emitter`?